src/yunohost/domain.py

```diff
@@ -61,7 +61,10 @@ def domain_list(exclude_subdomains=False):
                 continue
         result_list.append(domain)
 
-    return {'domains': result_list}
+    return {
+        'domains': result_list,
+        'default': _get_maindomain()
```

Comment:
```suggestion
        'main': _get_maindomain()
```

The semantic "main" should be kept (though iirc we kind of use "default domain" in places in the webadmin views but in cli we have for example `yunohost tools main-domain` etc... )